Gopkg.toml

```diff
@@ -1,25 +1,3 @@
-required = [
-    "github.com/emicklei/go-restful",
-    "k8s.io/client-go/plugin/pkg/client/auth/gcp", # for development against gcp
-    "k8s.io/code-generator/cmd/deepcopy-gen", # for go generate
-    "sigs.k8s.io/controller-tools/cmd/controller-gen", # for crd/rbac generation
```

Comment:
We need this to generate crds. The same holds for `deepcopy-gen`. Since they are not imported anywhere in the code we build by default, it's removed by dep. Please, put at least those two back.

Reply:
I also added back the commented version for integration.